modules/light-clients/08-wasm/client/cli/tx.go

```diff
@@ -47,3 +47,41 @@ func newPushNewWasmCodeCmd() *cobra.Command {
 
 	return cmd
 }
+
+// newUpdateWasmCodeId returns the command to create a UpdateWasmCodeId transaction
+func newUpdateWasmCodeId() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-wasm-code-id [client-id] [code-id]",
+		Short: "Updates wasm code id for a client",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			clientId := args[0]
+			codeId, err := transfertypes.ParseHexHash(args[1])
+
+			if err != nil {
+				return err
+			}
+
+			msg := &types.MsgUpdateWasmCodeId{
+				ClientId: clientId,
+				CodeId:   codeId,
+				Signer:   clientCtx.GetFromAddress().String(),
+			}
+
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
```

Comment:
This is a redundant validation since it is already called in the GenerateOrBroadcastTxCLI function.

Reply:
It applies to `newPushNewWasmCodeCmd` CLI command as well.

Reply:
Removed